lib/robot.js

```diff
@@ -25,6 +25,25 @@ class Robot {
     });
   }
 
```

Comment:
I wonder if we should only JSDoc those things that are actually Probot API functionality we want people to use like `on` or `route`

If you look just above at L22, that function is also undocumented.

@bkeepers thoughts?